k8s/base/settings.env

```diff
@@ -0,0 +1,2 @@
+ARCHIVE_URL=https://archive-api.lco.global/frames/
```

Comment:
This file should be moved to `k8s/envs/local/settings.env`

Reply:
Does this need to be in local settings if this is the default value for all deploys and you only have to override it in staging?

Reply:
If you'd like these to be the default, then you have to inline them using the `literal` form:

```yaml
configMapGenerator:
  - name: env
    literals:
      - ARCHIVE_URL=https://archive-api.lco.global/frames/
      - ETC=xxx
```

Reason being, any non-yaml files are not copied around by `kpt pkg get ...`

Reply:
Ok. Will do.